node/node.go

```diff
@@ -104,6 +104,7 @@ type Node struct {
 	Consensus              *consensus.Consensus // Consensus object containing all Consensus related data (e.g. committee members, signatures, commits)
 	BlockChannel           chan *types.Block    // The channel to send newly proposed blocks
 	ConfirmedBlockChannel  chan *types.Block    // The channel to send confirmed blocks
+	BeaconBlockChannel     chan *types.Block    // The channel to send beacon blocks for non-beaconchain nodes
```

Comment:
why do we need this? We do not propose new beacon blocks for regular nodes

Reply:
this is the channel to send the received block to beacon syncing program; it's not the same as the channel in consensus process